tools/hlint.sh

```diff
@@ -35,6 +35,14 @@ while getopts ':f:m:' opt
      esac
 done
 
+ret=$?
```

Comment:
If someone who actually knows shellscript has a better suggestion, please let me know :)

Reply:
Usually fighting bash to get nice error messages is not very fruitful and leaves error handling to the script writer. So I prefer it if we just add `set -euo pipefail` to all the scripts. You still have the option to error check and print nice errors, you just have to use `if ! <cmd>` instead of `if [ $? -ne 0 ]`. This way we don't have to worry about every line that is added to the bash script failing and error getting lost.